From 1f273feb856bac2acfa4b68b1ec4ae97387e0bb6 Mon Sep 17 00:00:00 2001
From: Haihua Hu <jared.hu@nxp.com>
Date: Tue, 27 Sep 2016 09:07:29 +0800
Subject: [PATCH] [MMFMWK-7364][imx8dv] fix qmlgltest can not run on FB backend

Add head fb backend files in gstgldisplay and gstglwindow, it was
missed.

Upstream-Status: Inappropriate [i.MX specific]

Signed-off-by: Haihua Hu <jared.hu@nxp.com>
---
 gst-libs/gst/gl/gstgldisplay.c | 3 +++
 gst-libs/gst/gl/gstglwindow.c  | 3 +++
 2 files changed, 6 insertions(+)

diff --git a/gst-libs/gst/gl/gstgldisplay.c b/gst-libs/gst/gl/gstgldisplay.c
index 0213d60..86c1e92 100644
--- a/gst-libs/gst/gl/gstgldisplay.c
+++ b/gst-libs/gst/gl/gstgldisplay.c
@@ -69,6 +69,9 @@
 #include <gst/gl/egl/gstgldisplay_egl.h>
 #include <gst/gl/egl/gsteglimagememory.h>
 #endif
+#if GST_GL_HAVE_WINDOW_FB
+#include <gst/gl/fb/gstgldisplay_fb.h>
+#endif
 
 GST_DEBUG_CATEGORY_STATIC (gst_context);
 GST_DEBUG_CATEGORY_STATIC (gst_gl_display_debug);
diff --git a/gst-libs/gst/gl/gstglwindow.c b/gst-libs/gst/gl/gstglwindow.c
index 7b20ff4..5a90021 100644
--- a/gst-libs/gst/gl/gstglwindow.c
+++ b/gst-libs/gst/gl/gstglwindow.c
@@ -58,6 +58,9 @@
 #if GST_GL_HAVE_WINDOW_EAGL
 #include "eagl/gstglwindow_eagl.h"
 #endif
+#if GST_GL_HAVE_WINDOW_FB
+#include "fb/gstglwindow_fb_egl.h"
+#endif
 #if GST_GL_HAVE_WINDOW_DISPMANX
 #include "dispmanx/gstglwindow_dispmanx_egl.h"
 #endif
-- 
2.9.0.GIT

